src/queries/index.ts

```diff
@@ -401,28 +401,24 @@ function getAssetInfo (queryOptions, params) {
 /**
  * getPeers
  * @param {Object} queryOptions
- * @param {Object} args
- * @property {Object[]} args.peersList
  * @link https://iroha.readthedocs.io/en/latest/develop/api/queries.html#get-peers
  */
-function getPeers (queryOptions, { peersList }) {
+function getPeers (queryOptions) {
   return sendQuery(
     queryOptions,
     queryHelper.addQuery(
       queryHelper.emptyQuery(),
       'getPeers',
-      {
-        peersList
-      }
+      {}
```

Comment:
If this argument mandatory, maybe better to set default value in addQuery method?

Reply:
But I think we just can add new task to fix it

Reply:
good idea, done